apps/condo/domains/common/components/BaseSearchInput/index.tsx

```diff
@@ -124,8 +167,16 @@ export const BaseSearchInput = <S extends string>(props: ISearchInput<S>) => {
     )
 
     const options = useMemo(
-        () => data.map((option) => renderOption(option, restSelectProps.value)),
-        [data, fetching, restSelectProps.value],
+        () => {
+            const dataOptions = data.map((option, index) => renderOption(option, value, index))
+
+            return fetching ? [...dataOptions, (
```

Comment:
This way it would be a bit more readable
```
            if (!fetching) return dataOptions

            return [
                ...dataOptions, 
                (
                    <Option key={'loader'} value={null} disabled>
                        <Loader style={SELECT_LOADER_STYLE}/>
                    </Option>
                )
            ]
```